arviz/data/inference_data.py

```diff
@@ -11,19 +11,53 @@
 
 
 class InferenceData:
-    """Container for accessing netCDF files using xarray."""
+    """Container for inference data storage using xarray.
+
+    For a detailed introduction on ``InferenceData`` objects and its usage, see
```

Comment:
For a detailed introduction to ``InferenceData`` objects and their usage, see

Reply:
Thanks for all the corrections!